File: wlr/OGUR/OGUR/Factory/AnimatedTextureFactory.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using Microsoft.Xna.Framework.Content;
using OGUR.Sprites;

namespace OGUR.Factory
{
    internal class AnimatedTextureFactory
    {
        public static AnimatedTexture Create(SpriteType type, int x, int y)
        {
            var sprite = new AnimatedTexture();
            sprite.LoadContent(type);
            sprite.SetPosition(x, y);
            return sprite;
        }
    }
}

File: port-workspace/spx-ported/src/com/spx/sprites/AnimatedTextureFactory.java
package com.spx.sprites;import com.spx.wrapper.*;import java.util.*;    public class AnimatedTextureFactory
    {
        public static AnimatedTexture Create(int type, int x, int y)
        {
            AnimatedTexture sprite = new AnimatedTexture();
            sprite.LoadContent(type);
            sprite.SetPosition(x, y);
            return sprite;
        }
    }
